package-lock.json

```diff
@@ -1,8450 +1,8 @@
 {
   "name": "prometheus-api-metrics",
   "version": "3.2.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
Looks like the package-lock got changed to v1, can you revert this change?

Reply:
@MatanAvneri do you think its safe to change it manually?

Reply:
No, it should be only done by `npm` but I don't think it should be related to this MR anyway

Reply:
so do you want me to revert all package-lock changes?

Reply:
reverted, here is the workflow of this commit:
https://github.com/ilanKushnir/prometheus-api-metrics/actions/runs/895185399